### ISSUE TITLE
Europeana module: use the API instead of the SPARQL endpoint

### ISSUE THREAD

**Issue by lyndonnixon** [linkedtv/wp2#29]
Hi, we should agree and finalise the means to enrich TKK content with Europeana items. I am keen with Lotte to prototype something in the next 3 weeks that could be part of the EUScreenXL presentation.

We know that http://www.linkedtv.eu/wiki/index.php/Europeana_Module is OK but not perfect, not only because we need less constrained queries (e.g. "Hessel Martena" SPARQL query works once less constrained) but also because the SPARQL endpoint is very slow (side note: It seems the mediaURLs you retrieve need additional parameters to actually resolve in the browser)

We also know CWI has some implementation using the API but no information about it. Could we get this information, so that we can test its results and if it is  a basis for further work?

**Comment by rtroncy** [comment]
@lyndonnixon We should clearly switch to the API instead of the SPARQL endpoint to get enrichments from Europeana. This work will start post-ISWC and should be finished before the 3rd year review.

**Comment by rtroncy** [comment]
Tagging @michielhildebrand and @lottebelice

**Comment by lottebelice** [comment]
What I'd want for the EUscreen conference  is to show a version of LinkedCulture in which we've enriched the content with EUscreen content that is findable through Europeana [1]. When I search on the ET on 'euscreen' or part of a title that is on Europeana from an EUscreen partner ('delphine' [2]), I get nothing.

However, looking at the timeline for this issue (post ISWC up to Y3 review), @lyndonnixon and I need to find a workaround for this. @michielhildebrand Any suggestions, based on your Europeana API experience?

[1] http://europeana.eu/portal/search.html?query=PROVIDER%3A%22EUscreen+Project%22
[2] http://europeana.eu/portal/record/09202/EUS_3C0D0251EDED4918A3E4E02E4293F4C3.html?start=2&query=PROVIDER%3A%22EUscreen+Project%22&startPage=1&rows=24

**Comment by rtroncy** [comment]
@lottebelice You want to have the Europeana enrichments in the editor tool, right? At the moment, you get nothing because Jaap is solely calling Europeana via TVEnricher so this is normal. A quick workaround is for Jaap to simply call the Europeana API instead of the Europeana module of TVEnricher as an interim solution.

**Comment by lottebelice** [comment]
Sounds good! Now....how can I tag Jaap in this issue? If I type @jblom, nothing happens. @rtroncy Do you need to add him? Or should I add a separate issue to the ET Github issue list [1]? 

[1] https://github.com/linkedtv/editortool/issues/

**Comment by rtroncy** [comment]
Now it does, I have indeed added @jblom to the WP2 team, the largest LinkedTV team now :+1:

**Comment by michielhildebrand** [comment]
Most likely the Europeana API allows a filter on something like collection. The documentation should tell you. Jaap could configure a dimension in the editor tool to include this filter. We can also directly include it in a dimension of the culture application or add a dimension. Let us know if you need that.

**Comment by rtroncy** [comment]
The Europeana API documentation you need is at http://labs.europeana.eu/api/query/. The query syntax is simple. To realize @lottebelice use case, you will need to do a basic search + use the facet parameter (qf) with the value PROVIDER ... so something like: http://www.europeana.eu/api/v2/search.json?wskey=xxxx&query=Hessel+Martena&qf=PROVIDER:EUScreen+Project

**Comment by rtroncy** [comment]
@lyndonnixon has produced some code that has not been integrated
